packages/manager/apps/pci-workflow/src/components/new/Status.component.tsx

```diff
@@ -0,0 +1,53 @@
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { OdsChipAttribute } from '@ovhcloud/ods-components';
+import { OsdsChip } from '@ovhcloud/ods-components/react';
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+type StatusComponentProps = {
+  statusGroup: string;
+  status: string;
+};
+export default function StatusComponent({
+  statusGroup,
+  status,
+}: Readonly<StatusComponentProps>) {
+  const { t } = useTranslation('new');
+  const [chipAttribute, setChipAttribute] = useState<OdsChipAttribute>({
+    color: ODS_THEME_COLOR_INTENT.success,
```

Comment:
Why not initializing the state with `ODS_THEME_COLOR_INTENT.info` as it's the default value in the `switch` ?